app/src/main/java/com/example/plantsapp/data/firebase/entity/FirebasePlant.kt

```diff
@@ -9,20 +9,20 @@ data class FirebasePlant(
     val plantPicture: String = ""
 ) {
 
-    fun toPlant(): Plant {
+    fun toPlant(localImageUri: Uri?): Plant {
```

Comment:
Actually - probably you can store image , get it id and in FirebasePlant store link to this image as plantPicture plant